test/jdk/java/beans/Introspector/7084904/Test7084904.java

```diff
@@ -27,12 +27,11 @@
  * @bug 7084904
  * @summary Compares reflection and bean introspection
  * @library ..
- * @run main/othervm -Djava.security.manager=allow Test7084904
+ * @run main Test7084904
  * @author Sergey Malenkov
```

Comment:
The test below `Test4683761.java` removes the `@author` tag. Should it be removed here?

Reply:
This can be handled later in a more general cleanup task.